app/assets/javascripts/application.js

```diff
@@ -229,28 +229,32 @@ function filter_by_level(item){
 
   // Note that class names don't map to log level names (label-info == notice)
   if(level == 'info'){
-    $('.label-info').closest('tr').show();
-    $('.label-default').closest('tr').show();
-    $('.label-warning').closest('tr').show();
-    $('.label-danger').closest('tr').show();
+    $('.label-info.result-filter-tag').closest('tr').show();
+    $('.label-default.result-filter-tag').closest('tr').show();
```

Comment:
> When you say $('.label-{tag}.result-filter-tag'), it will give you DOM element which having both these CSS classes i.e label-{tag} and result-filter-tag.

That is exactly what I want.

Perhaps I should have provided more detailed example as it is not obvious if there is only one row in the table:

```html
<table>
  <thead>
    <tr>
      <th>filter by this column</th>
      <th>do not filter by this column</th>
    </tr>
  <thead>
  <tbody>
     <tr>
       <td><span class="label label-info result-filter-tag">value to filter - Info</span></td>
       <td><span class="label label-success">value I do not filter - Success</span></td>
     </tr>
     <tr>
       <td><span class="label label-warning result-filter-tag">value to filter - Warning</span></td>
       <td><span class="label label-success">value I do not filter - Success</span></td>
     </tr>
     <tr>
       <td><span class="label label-success result-filter-tag">value to filter - Success</span></td>
       <td><span class="label label-warning">value I do not filter - Warning</span></td>
     </tr>
     <!-- more rows to follow -->
  <tbody>
</table>
```

if, for example, `level == 'notice' ` in `filter_by_level` function, then I want to show all the rows that have in the first column label-info, label-warning, or label-danger (first two rows), therefore I want to include them in my selector. Leaving just `$('.result-filter-tag').closest('tr')` will return all the rows, which is not what I want.